### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by LeiyuanMa** [jiwoon-ahn/psa#8]
in train_cls.py,when the learning rate is set to 0.1,the loss is increasing,so I tried to set it to 0.01,it works,,so can you verify the learning rate in train_cls.py and train_aff.py?Thanks in advance!

**Comment by jiwoon-ahn** [comment]
That should work fine. Perhaps there is a problem while loading the pre-trained model. Could you check (https://github.com/jiwoon-ahn/psa/issues/1)?

**Comment by LeiyuanMa** [comment]
modelA and modelA1 doesn't make difference,for me the learning rate need to be set to 0.01,after train for a night,the loss of validation decreased to 0.79,it seems works.
By the way, in train_cls.py,line133 and line 142,the "if...else..."really confused me,I think I understand what you mean,in order to be more clear,could you revise it ?

**Comment by LeiyuanMa** [comment]
modelA and modelA1 doesn't make difference,for me the learning rate need to be set to 0.01,after train for a night,the loss of validation decreased to 0.079,it seems works.
By the way, in train_cls.py,line133 and line 142,the "if...else..."really confused me,I think I understand what you mean,in order to be more clear,could you revise it ?

**Comment by jiwoon-ahn** [comment]
The learing rate given in the tutorial is only for VGG-16.
It is not "if ... else" sentence, rather it is "for ... else" sentence. I don't see any problem in there.
But the code is not perfect. It does contain many typos, grammar errors, etc.The best way to fix it is to fork yours.

**Comment by LeiyuanMa** [comment]
Thanks for your reply,it works fine now.
